skyportal/tests/conftest.py

```diff
@@ -624,9 +624,11 @@ def public_groupuser(public_group, user):
 
 
 @pytest.fixture()
-def user2(public_group):
+def user2(public_group, public_stream):
```

Comment:
Users all should have stream access to the streams that their group's filters filter. In this case, `public_group` had a `Filter` that filtered Stream 1, but `user2` did not have that stream. Nevertheless, they were permitted to be in the group. Tests that instantiated the Filter and the user in the same session raised an accesserror because the access policy for filters requires that users accessing them have access to their streams. This fixes that.